projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -31,16 +31,6 @@ function wpcom_add_wpcom_menu_item() {
 	$parent_slug = 'wpcom-hosting-menu';
 	$domain      = wp_parse_url( home_url(), PHP_URL_HOST );
 
-	add_menu_page(
-		esc_attr__( 'All Sites', 'jetpack-mu-wpcom' ),
-		esc_attr__( 'All Sites', 'jetpack-mu-wpcom' ),
-		'manage_options',
-		'https://wordpress.com/sites',
-		null,
-		'dashicons-arrow-left-alt2',
-		0
-	);
```

Comment:
If I understand correctly this should remove the 'all sites' link from proxied users using the nav redesign as well.

When you tested this did include build/sync/enabling dev for jetpack-mu-wpcom as well?

Reply:
I followed your suggestions on slack, so this is not needed anymore, but TIL 😉